### ISSUE TITLE
Visual Selection with mouse does not work with multiple tabs

### ISSUE THREAD

**Issue by Bilalh** [neovim/neovim#2382]
with `mouse=a`.    

If the there more then one tab  visual section with the mouse breaks.  
Works correctly in vim. 

No other config in .nvimrc/.vimrc. 

neovim version
```
NVIM 0.0.0-alpha+201504081227 (compiled Apr  8 2015 18:43:53)
Commit: e1bac3b84090d07afa669a8b79816c28813c605e
```

vim version 
```
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Apr 21 2014 14:54:22)
MacOS X (unix) version
Included patches: 1-258
```

**Comment by mhinz** [comment]
Confirmed.

Steps for reproducing:

1) `:set mouse=a`
2) `:tabnew` some times
3) Now something interesting happens: you can still use visual selection, if you switch between tabs via `gt`/`gT`, but not anymore if you clicked on a tab at the top.

I'll investigate.

**Comment by Hettomei** [comment]
Hi,

It is same as issue #2119 , should I close it ?
I confirm : You have to click on tab but gt gT works (with ```nvim -u NONE```).

**Comment by Hettomei** [comment]
And if after it breaks you do

```
:mksession
:q

nvim -S
```

mouse works well.

**Comment by Pyrohh** [comment]
@mhinz can this be closed since #2382 was merged?

**Comment by mhinz** [comment]
@Pyrohh Oh, yes. Thanks.
